stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py

```diff
@@ -50,6 +50,13 @@ def default_get(self, fields):
         string="Barcode format",
         default=lambda self: self.env.company.barcode_default_format,
     )
+    barcode_report = fields.Many2one(
+        comodel_name="ir.actions.report",
+        string="Report to print",
+        domain=[("is_barcode_label", "=", True)],
+        default=lambda self: self.env.company.barcode_default_report,
+        required=True,
```

Comment:
> @joao-p-marques That was happening by the test of stock_picking_product_barcode_report_secondary_unit, because I introduce a needed field on the wizard creation. I changed it in a new commit on this PR

Ok, looks good now, but could you avoid the `required` or set a more resilient default? Maybe, if the default is not set on a company level, you could assign one? 

That way, you could avoid changing the other module, and further changes for the same reason...

Reply:
This field has to be required, because is the format of the labels printed. So I make the default as you said